shell/browser/api/electron_api_web_contents.cc

```diff
@@ -1505,7 +1505,8 @@ content::JavaScriptDialogManager* WebContents::GetJavaScriptDialogManager(
 }
 
 void WebContents::OnAudioStateChanged(bool audible) {
-  Emit("-audio-state-changed", audible);
+  DCHECK_EQ(audible, IsCurrentlyAudible());
+  Emit("audio-state-changed", audible);
```

Comment:
We don't yet have helpers to make emitting custom event objects ergonomic, so this will have to be the somewhat awkward:
```suggestion
  gin::Handle<gin_helper::internal::Event> event =
      gin_helper::internal::Event::New(isolate);
  v8::Local<v8::Object> event_object = event.ToV8().As<v8::Object>();

  gin::Dictionary dict(isolate, event_object);
  dict.Set("audible", audible);
  EmitWithoutEvent("audio-state-changed", event);
```